src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -640,6 +640,19 @@ public sealed class UnscopedRefAttribute : Attribute
     }
 }";
 
+        protected const string RefSafetyRulesAttributeDefinition =
+@"namespace System.Runtime.CompilerServices
+{
+    public sealed class RefSafetyRulesAttribute : Attribute
```

Comment:
should this definition use AttributeUsageAttribute?

Reply:
This is just an example definition that is included as source in some tests so it didn't seem necessary to use `[AttributeUsage]`.